spec/acceptance/cli/update_spec.rb

```diff
@@ -20,8 +20,8 @@
 
   after do
     in_test_directory do
-      `gem uninstall dummy -v 1.0.1`
-      `gem uninstall dummy2 -a`
+      %x{gem uninstall dummy -v 1.0.1}
+      %x{gem uninstall dummy2 -a}
```

Comment:
ditto

Reply:
Fixed in 2d022fcbd3f0404f07e570ca78d7e256eaddebcb